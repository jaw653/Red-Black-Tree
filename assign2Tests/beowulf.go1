r
f grendel
f beowulf
f "miley cyrus"
f shields
d shields
f shields
d shields
f shields
r
