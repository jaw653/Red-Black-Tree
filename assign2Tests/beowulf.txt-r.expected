Words/Phrases: 33360
Nodes: 5936
Minimum depth: 10
Maximum depth: 15
Frequency of grendel: 73
Frequency of beowulf: 156
Frequency of miley cyrus: 0
Frequency of shields: 2
Frequency of shields: 1
Frequency of shields: 0
Words/Phrases: 33358
Nodes: 5935
Minimum depth: 10
Maximum depth: 15
